pkgs/by-name/os/ostui/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromSourcehut,
+  pkg-config,
+  mpv-unwrapped,
+}:
+buildGoModule (finalAttrs: {
+  pname = "ostui";
+  version = "1.0.4";
+
+  src = fetchFromSourcehut {
+    owner = "~ser";
+    repo = "ostui";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-efX19jkJnXyO4iuY2EZqhtLJZ7R/Q2JQZf72gyLgY8k=";
+  };
+
+  vendorHash = "sha256-Vmjd0bbeR9+PZCjh1cczE5MWeH5PDVE6obJLmV0wCLQ=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    mpv-unwrapped
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${finalAttrs.version}"
+  ];
+
+  env = {
+    CGO_ENABLED = "1";
+  };
```

Given the tests only fail on darwin I'd like to suggest the following:

```suggestion
  };
  
  doCheck = !stdenv.hostPlatform.isDarwin;
```

---

I've looked upstream, and indeed the checks are not that significant, and I think we can skip the check on Darwin. Perhaps in the future, it might somehow start working properly, but as of now, I think your approach is the most appropriated.